code/game/machinery/computer/crew.dm

```diff
@@ -296,14 +296,20 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 			results[++results.len] = entry
 			continue
 
-		// BUBBERSTATION EDIT BEGIN: Add DNR status
+		// BUBBERSTATION EDIT BEGIN: Add DNR status, proteans death state.
 		// If sensors are above living tracking, set DNR state
 		if (sensor_mode >= SENSOR_LIVING)
 			entry["is_dnr"] = tracked_human.get_dnr()
-
-		// Current status
-		if (sensor_mode >= SENSOR_LIVING)
-			entry["life_status"] = tracked_living_mob.stat
+			// Current status
+			if(!isprotean(tracked_human))
+				entry["life_status"] = tracked_living_mob.stat
+			else
+				// Check if protean is stuck in suit
+				var/obj/item/organ/brain/protean/brain = tracked_human.get_organ_slot(ORGAN_SLOT_BRAIN)
+				if(brain.dead)
+					entry["life_status"] = 4
+				else // If brain not dead/no brain then handling as usual
+					entry["life_status"] = tracked_living_mob.stat
```

Comment:
```suggestion
			// Current status
			if(!isprotean(tracked_human))
				entry["life_status"] = tracked_living_mob.stat
			else
				// Check if protean is stuck in suit
				var/obj/item/organ/brain/protean/brain = tracked_human.get_organ_slot(ORGAN_SLOT_BRAIN)
				entry["life_status"] = brain?.dead ? DEAD : tracked_living_mob.stat // If brain not dead/no brain then handling as usual
		// BUBBERSTATION EDIT END
```
Slightly cleaner
- `?.` will ensure that if `get_organ_slot` returned null we won't runtime trying to access `dead` on it
- changed the raw `4` to be the `DEAD` define for stat
- we can use a ternary operator to turn the `if` into a single line